runhouse/rns/login.py

```diff
@@ -113,10 +116,11 @@ def login(
         configs.download_and_save_defaults()
         # We need to fresh the RNSClient to use the newly loaded configs
         rns_client.refresh_defaults()
-    elif upload_config:
+    if upload_config:
```

Comment:
should we enable uploading (after downloading and saving differences) for anything that might be set locally that hasn't been reflected in den yet? or should these be mutually exclusive?

Reply:
I originally tried to make it so it would offer to upload the new merged config, but I think it's too complicated to explain. I think mutually exclusive is fine for now.